app/javascript/elm/src/Main.elm

```diff
@@ -285,10 +287,16 @@ update msg model =
 
         UpdateSessions value ->
             let
+                decoder =
+                    Decode.map2 FetchedSessions
```

Comment:
What if you do something like `Decode.map2 Tuple.pair` and then

```
                (sessions, amount) =
                    Decode.decodeValue (Decode.list Data.Session.decoder) value	                    Decode.decodeValue decoder value
                        |> Result.withDefault ([], 0)
```
That way you don't need to name the type.

Rationale: I feel that the `FetchedSession` type is a bit forced in this case

Reply:
I agree, thanks! I was looking for a shorter way to do this :)